Utilities/aliceHLTwrapper/src/WrapperDevice.cxx

```diff
@@ -174,7 +174,7 @@ void WrapperDevice::Run()
         mMaxTimeBetweenSample=sampleTimeDiff;
     }
     mLastSampleTime=duration.count();
-    if (duration.count()-mLastCalcTime>fLogIntervalInMs) {
+    if (duration.count()-mLastCalcTime>1000) {
```

Comment:
is there no GetInterval() function? Why did you choose 1000? ( I believe we should have a named variable for constants. )